server/app/repository.py

```diff
@@ -379,6 +393,33 @@ def read_station_secret(self, station_id: StationId) -> Optional[bytes]:
             return None
         return bytes(row["secret"])
 
+    def user_role_to_enum(self, role: str) -> UserRole:
+        """Converts string to UserRole enum"""
+        for u in UserRole:
```

Comment:
In ```utils.py``` you have ```first``` function.

```python
return first(lambda u: u.name == role.upper(), UserRole)
```
but it may return None then you need test for it.

Reply:
Updated as suggested. Also testing if None is returned.

On a related note, it's super confusing that we have several utils.py. We should rename rename one of them one day. One way to deconflict the names would be to rename server/tests/utils.py to server/tests/test_utils.py.

Reply:
Ok, good suggestion.

Reply:
Renamed.